guides/source/active_job_basics.md

```diff
@@ -619,14 +619,21 @@ class MoneySerializer < ActiveJob::Serializers::ObjectSerializer
   def deserialize(hash)
     Money.new(hash["amount"], hash["currency"])
   end
+
+  private
+    def klass
+      Money
+    end
```

Comment:
Based on the API docs, it looks like the addition of `klass` makes `serialize?` unnecessary since `ObjectSerializer`'s implementation is `argument.is_a? klass`.

I like the idea of going with the simpler API, but let's remove `serialize?` and move its comment to explain why `klass` is needed.

Reply:
How does this look?

41f3bd0837934eddd9f3fb918affde52c96aac44